packages/block-editor/src/components/block-popover/index.js

```diff
@@ -74,6 +77,13 @@ function BlockPopover(
 		};
 	}, [ selectedElement ] );
 
+	const { isZoomOut } = useSelect(
+		( select ) => ( {
+			isZoomOut: unlock( select( blockEditorStore ) ).isZoomOut(),
+		} ),
+		[]
+	);
```

Comment:
Nit: because `isZoomOut` is a simple boolean, you can simplify this to:
```js
const isZoomOut = useSelect( ( select ) => unlock( select( blockEditorStore ) ).isZoomOut(), [] );
```